.gitlab/check.yml

```diff
@@ -0,0 +1,31 @@
+.check_skip_rust:
+  before_script:
+    - |-
+      if echo "$CI_COMMIT_TITLE" | grep -q "ci-skip-rust"; then
+        echo "Skipping rust tests due to commit title"
+        exit 0
+      fi
+
+.check_skip_integration_tests:
+  before_script:
+    - |-
+      if echo "$CI_COMMIT_TITLE" | grep -q "ci-skip-integration-tests"; then
+        echo "Skipping integration tests due to commit title"
+        exit 0
+      fi
+
+cargo-clippy:
```

Comment:
We don't check the docs anymore?

Reply:
Nice catch

Reply:
Nice catch

Reply:
I still don't see this step (?)

Reply:
It was in the test section, moved it to check